server.js

```diff
@@ -62,4 +62,10 @@ if (fs.existsSync(legacySecretsPath)) {
 }
 export const server = new Server(config)
 
+process.on('SIGTERM', async () => {
+  console.log('SIGTERM received, shutting down...')
+  await server.stop()
+  process.exit(0)
+})
+
```

Comment:
Thanks.
I agree we should handle this, but I think this code has an issue: `process.exit()` doesn't wait for the event loop to clear, so `server.stop()` probably won't resolve cleanly.
There's a blog post about this here: https://kostasbariotis.com/why-you-should-not-use-process-exit/
I've not tested it, but I think we should be able to just remove the `process.exit()` call and let node naturally exit once `server.stop()` resolves. Can you have a look at this?

Reply:
Good to know, changed and tested it!